### DISCUSSION TITLE
Cannot assign null to property App\Http\Livewire\RegisterTeam::$password of type string

### DISCUSSION THREAD

**Discussion by dejagersh** [livewire/livewire#3312] [1. Help]
I have a registration form like such: 

```
class RegisterTeam extends Component
{
    ...

    public string $password = '';
```

We just got a Sentry saying that it tried to sync the value `null` to `$password`, which is not valid. The input field looks like this (with the classes omitted):

`<input type="password" required>`

Are `null` values something to take into account? I thought they always would be empty strings? In what situation does it sync to `null`? I tried to replicate it on many browsers but I'm unable to do so.

The request was made on a `Mobile Safari 14.1.1` browser.

One thing that I find odd, is that the payload of the request contains two updates for the same field:

```
{
    "updates": [
        {
            "type": "syncInput",
            "payload": {
                "name": "password",
                "value": null
            }
        },
        {
            "type": "syncInput",
            "payload": {
                "name": "password",
                "value": "Gavdes-6wowvy-javkaq"
            }
        }
    ]
}
```

Is that normal? I'm not too knowledgeable on the inner workings of Livewire, but it seems weird to me.

**EDIT:**

I just managed to reproduce it. When letting LastPass fill in the field, it breaks. What I think is happening:

1. LastPass clears the field.
2. LastPass fills in the field with the password coming from LastPass.

And I did some more digging and it happens _only_ on Safari. Filing in the password using LastPass on Chrome works fine. Looking at the password, this happened with Keychain.

I would love to have some guidance on how to avoid this. Should I just _always_ make my fields nullable?

https://github.com/livewire/livewire/issues/1081 This seems to be the same issue. I tried regular auto filling the `$name` field as well and it also triggers a 500 error. It should be fixed according to that discussion, but it seems it's not. Am I missing something?

P.S. I am using Livewire 2.5.3

P.S. I made a discussion here because I was doing something wrong, but now it _feels_ like a bug in Livewire. If it is, let me know if I should make an issue rather than having it here in Discussions

**Comment by joshhanley** [comment]
@thejager safari autofill is a common issue with Livewire. There have been a few attempts to fix it, if you have a look back at issues and PRs. But each time something still doesn't quite work right.

Not sure what you can do, maybe some of the previous threads have some ideas.
